docs/api-docs/management-apis/contextual-filters.md

```diff
@@ -0,0 +1,310 @@
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#requestrunner) 
+// replace link
+
+The response will contain all of the global filters currently configured on your store. 
+
+```json
+{
+   "data": [
+       {
+           "id": "Y2F0ZWdvcnk=",
+           "display_name": "Category",
+           "type": "category",
+           "items_to_show": 15,
+           "collapsed_by_default": false,
+           "display_product_count": true,
+           "is_enabled": true
+       },
+       {
+           "id": "U2l6ZQ==",
+           "display_name": "Size",
+           "type": "product",
+           "display_product_count": true,
+           "collapsed_by_default": false,
+           "items_to_show": 10,
+           "sort_by": "alpha",
+           "is_enabled": true
+       },
+       {
+           "id": "cHJpY2U=",
+           "display_name": "Price",
+           "type": "price",
+           "collapsed_by_default": false,
+           "is_enabled": true
+       }
+       ...
+   ],
+   "meta": {}
+}
+```
+The global filters returned from completing a `GET` request should match the filters you see in the control panel under **Storefront Filters**. This set of filters is what you currently have configured on your store. 
+
+To access **Storefront Filters** in the control panel, go to **Products** > **Product Filtering**. 
+
+![Products Filters](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/contextual-filters-01.png "Products Filters")
+
+## View available filters
+
+Available filters are the filters you can configure on the global level. To see all available configurable filters, send a `GET` request to [`/v3/settings/search/filters/available`](insert-link-here). To retrieve a list of filters available on a given category, add `category_id` as a query parameter.
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#requestrunner) // replace link
+
+The response will contain all of the filters available for configuration. 
+
+```json
+{
+    "data": [
+        {
+            "id": "YnJhbmQ=",
+            "type": "brand",
+            "name": "Brand",
+            "product_count": 3
+        },
+        {
+            "id": "cmF0aW5n",
+            "type": "rating",
+            "name": "Rating",
+            "product_count": 1
+        },
+        {
+            "id": "cHJpY2U=",
+            "type": "price",
+            "name": "Price",
+            "price_range_min": 9.95,
+            "price_range_max": 225
+        },
+        ...
+    ],
+    "meta": {}
+}
+```
```

Comment:
🍹 maybe 1 sentence about "what this means" for this particular response.

e.g. "This means I have a Brand, Rating, and Price filter available for configuration. I can pick from these 3 filters when I am choosing which filters to make active. In reality, there will be more filters."